draft-looker-jwt-cwt-status-list.md

```diff
@@ -142,7 +143,54 @@ The following rules apply to validating the "status_list" (status list) claim
 
 2. The claim value object MUST contain a "typ" (type) attribute with a string based value that represents the type of status list referenced. The value MUST be equal to that of the "typ" attribute in the "status" claim for the token who's status is being validated.
 
-3. The claim value object MUST contain a "lst" (list) attribute with a string based value that represents the status values for all the tokens it conveys statuses for. The value MUST be a base64 encoded string using RFCXXX containing a GZIP compressed octet string {{RFC1952}}.
+3. The claim value object MUST contain a "lst" (list) attribute with a string based value that represents the status values for all the tokens it conveys statuses for. The value MUST be computed using the algorithm described in [](#jwt-status-list-claim-encoding).
+
+### Status List Encoding {#jwt-status-list-claim-encoding}
+
+Each status of a credential must be representable by a bit-size of 1,2,4, or 8. This means 2,4,16, or 256 possible statuses for a credential depending on the bit-size of choice. This limitation was introduced to reduce the amount of bit manipulation necessary to one byte for every operation and thus keeping implementations simpler and less error prone.
```

Comment:
If #20 is merged ahead will need to update this PR with the required language alignment e.g credential -> subject token

Reply:
Updated with referenced token by paul